saleor/plugins/avatax/plugin.py

```diff
@@ -421,6 +422,24 @@ def fetch_taxes_data(self, previous_value):
         get_cached_tax_codes_or_fetch(self.config)
         return True
 
+    @classmethod
+    def validate_authentication(cls, plugin_configuration: "PluginConfiguration"):
+        conf = {
+            data["name"]: data["value"] for data in plugin_configuration.configuration
+        }
+        url = urljoin(get_api_url(conf["Use sandbox"]), "utilities/ping")
+        response = api_get_request(
+            url,
+            username_or_account=conf["Username or account"],
+            password_or_license=conf["Password or license"],
```

Comment:
Do we use strings with spaces as keys in the configuration?

Reply:
Unfortunately for Avalara, yes. 
This should be changed but imo not in this PR.

Reply:
Yes I agree it's a separate task but I thought we fixed it already, but now I see this was done properly only for Adyen.